actions/position.js

```diff
@@ -4,6 +4,8 @@ import { update } from './overview';
 export const UPDATE_POSITION = 'UPDATE_POSITION';
 export function updatePosition(latitude, longitude) {
   return (dispatch) => {
+    if (!latitude || !longitude) return;
```

Comment:
@smaclell this was added to prevent that issue I was facing yesterday

Reply:
Excellent. I think we are seeing some issues on the beta with this.

Reply:
This has been fixed on master and should not be needed.